src/app/dialogs/confirm-delete-dialog/confirm-delete-dialog.component.html

```diff
@@ -0,0 +1,21 @@
+<dialog #confirmDeleteDialog id="confirm-delete-dialog">
+    <div class="dialog-header hazard-background"></div>
+    <p>Are you sure you want to delete this {{ maintenanceEvent ? "maintenance event" : "incident" }}:</p>
+    <p><em>{{ incident.displayName }}</em></p>
+    <p>This operation cannot be undone.</p>
+    <div class="dialog-controls">
+        <button
+            type="button"
+            (click)="userConfirms()"
+            class="dangerous"
+        >
+            <fa-icon [icon]="icons.delete"></fa-icon><span>Delete</span>
+        </button>
+        <button
+            type="button"
+            (click)="userCancels()"
+        >
+            <fa-icon [icon]="icons.delete"></fa-icon><span>Cancel</span>
+        </button>
+    </div>
+</dialog>
```

Comment:
```suggestion
</dialog>

```

Nitpick: Missing new line.